### PR TITLE
Create standard issue templates

### PR DESCRIPTION
**Pull request by rtluu**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-issue.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- Test User _(if applicable)_:
+
+## Steps to Reproduce
+- URL:
+- Test User _(if applicable)_:
+
+## Desired behavior
+_Describe in detail what was supposed to happen; reference existing Github issue that contains a solution if possible_
+
+## Acceptance Criteria
```
Comment (gunsch) [inline .github/ISSUE_TEMPLATE/bug-issue.md]: what's the difference between "Acceptance Criteria" and "Definition of Done"?

File: .github/ISSUE_TEMPLATE/bug-issue.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- [ ] _What change needs to be made?_
+
+## Definition of Done
+- [ ] **Attached to a Milestone** (when will this be completed?)
+- [ ] **Attached to Epic** (what body of work is this a part of? possibly `Ongoing Maintenance`)
+- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
+- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `call center`, `research`, `accessibility`, `content`)
+- [ ] **Labeled with `Bug`**
```
Comment (gunsch) [inline .github/ISSUE_TEMPLATE/bug-issue.md]: can you put this at the top? (here and the other templates)


```
---
labels: bug
---
```

https://help.github.com/en/articles/manually-creating-a-single-issue-template-for-your-repository

Comment (rtluu) [reply .github/ISSUE_TEMPLATE/bug-issue.md]: I just did this for the `bug` issue. Zenhub automatically tags Epic tickets with the `Epic` label so we should be good there. As far as the standard issue template, I don't think we want to add any front matter there cause they presumably will all be different

File: .github/ISSUE_TEMPLATE/bug-issue.md (hunk 1 above)
Comment (rtluu) [reply .github/ISSUE_TEMPLATE/bug-issue.md]: the first answer here in this Stack Exchange thread i think does a great job of laying out the differences https://pm.stackexchange.com/questions/18430/what-is-the-difference-between-acceptance-criteria-and-the-definition-of-done

File: .github/ISSUE_TEMPLATE/epic-issue.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+As a _____, I need _____ so I can _____.
+
+## Hypothesis/(es)
+**If** _we make this change_ **then** _we expect this to happen_.
+
+## Goals
+_What outcome(s) do we want to see?_ 
+
+## KRs
+_Which VSP KRs does this body of work push forward?_ (`Intake & Onboarding`, `Automation + Instrumentation`, `Support`, `Capacity-building`)
```
Comment (gunsch) [inline .github/ISSUE_TEMPLATE/epic-issue.md]: KRs or Objectives? I think KRs make more sense here but it seems like you listed Objectives as examples

File: .github/ISSUE_TEMPLATE/bug-issue.md (hunk 1 above)
Comment (gunsch) [reply .github/ISSUE_TEMPLATE/bug-issue.md]: based on that description, maybe reword from "What change needs to be made?" to something like "Describe the desired working behavior" ?

File: .github/ISSUE_TEMPLATE/epic-issue.md (hunk 3 above)
Comment (gunsch) [reply .github/ISSUE_TEMPLATE/epic-issue.md]: great, let's do that then?

**Review by gunsch — CHANGES_REQUESTED** [review]
